frontend/src/components/candidates/candidateBox.js

```diff
@@ -119,28 +119,61 @@ class CandidateBox extends Component {
             <p>
               <b>Additional Comments:</b> {candidate.additionalComments}
             </p>
-            <h5 className="text-info pt-3">Interview Information</h5>
-            <p>
-              <b>Average Score: </b> {this.state.avgInterviewScore}
-            </p>
-            {this.state.interviews.map(interview => (
-              <p key={interview._id}>
-                <b>Interviewer: </b>
-                {interview.interviewer_name}
-                {interviewGetCategorySection(interview) !== null ? (
-                  <p>
-                    <b>Category given:</b> {interviewGetCategorySection(interview).response.text}
-                  </p>
-                ) : null}
-              </p>
-            ))}
+            <Row>
+              <Col md={6}>
+                <h4 className="text-info pt-3 pb-2">Interview Information</h4>
+              </Col>
+              <Col md={6} className="text-right">
+                <Button
+                  outline
+                  color="primary"
+                  className="mt-3"
+                  onClick={() => this.props.handleShowAllInterviews(candidate._id)}
+                >
+                  Show Interviews
+                </Button>
+              </Col>
+            </Row>
+            <div className="round-info-box">
+              <Table borderless className="seventy-percent-width">
+                <tbody>
+                  <tr>
+                    <td>
+                      <b>Average Score: </b>
+                    </td>
+                    <td>{this.state.avgInterviewScore}</td>
+                  </tr>
+                  {this.state.interviews.map(interview => (
+                    <tr key={interview._id}>
+                      <td>
+                        <b>Interviewer: </b>
+                      </td>
+                      <td>
+                        {interview.interviewer_name}
+                        {interviewGetCategorySection(interview) !== null ? (
```

Comment:
could do `{interviewGetCategorySection(interview) !== null && (...)}` instead for clarity

Reply:
@tko22 just this one left

Reply:
ok can you approve next time so we don't have to go back and forth, especially since these are nits

Reply:
ya sounds good

Reply:
thanks!